CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.12.0
+
+- Bump purchases-ios to 3.14.0 [Changelog here](https://github.com/RevenueCat/purchases-ios/releases/tag/3.14.0)
```

Comment:
I think the change in the init should be documented here as a breaking change. 
And the version number should be 2.0.0 if we're following semver. I think that's fine, this is an internal library, so there's really no problem in it being major. But we should still stick to semver. 

It might be confusing otherwise if you're making a release of something like `cordova-plugin-purchases`, you bump to the latest hybrid-common version as per the RELEASING.md instructions, then it stops working and you don't understand why

Reply:
Alternatively I can do a 3.14.1 fixing purchases-ios and have this be a minor. What do you think?

Reply:
Yes. It feels scary to do a major, but yes, it's not a big deal in this case. I of course had to break something with that iOS release lol